### ISSUE TITLE
term ids should have a iriorcurie range and should have a pattern that requires a prefix

### ISSUE THREAD

**Issue by turbomam** [microbiomedata/nmdc-schema#837]

**Comment by aclum** [comment]
Has this been addressed w/all the curie/identifier work that has happened over the summer?

**Comment by turbomam** [comment]
I don't think so, based on the linked issue. But we don't need both.

**Comment by turbomam** [comment]
No, I am [working around it](https://github.com/microbiomedata/nmdc-schema/blob/9e4c58099b2fe064b6b08d3e24cfaf36458cbefb/project.Makefile#L485-L494). @cmungall and the LinkML team know about the LinkML RDF "anyuri" problem, although I can't find a linkml/linkml issue.
